.github/workflows/build.yml

```diff
@@ -307,12 +307,13 @@ jobs:
           done
 
           # Compare and find stale files (exist in bucket but not in upload list)
-          comm -23 <(sort existing_files.txt) <(sort uploaded_root_files.txt) > stale_files.txt
+          # Exclude .blockmap files from deletion to preserve differential update capability
+          comm -23 <(sort existing_files.txt) <(sort uploaded_root_files.txt) | grep -v '\.blockmap$' > stale_files.txt
```

Comment:
I haven't been able to get differential updates to work yet, but we do need blockmaps at the root level to support them.